### PR TITLE
My version of Exercise 1

### PR DESCRIPTION
**Pull request by sgdiosdado**
Product of first basic html and css exercise.

### CODE REVIEW

File: Exercise-HTML-CSS-1/index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/styles.css">
+    <title>Girl Develop It! by SG</title>
+</head>
+<body>
+    <!-- Navegacion -->
+    <nav class="container-fluid navbar navbar-light bg-pink navbar-expand">
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/index.html]: Nice feature! I really like this

File: Exercise-HTML-CSS-1/index.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+                <li class="nav-item">
+                    <a class="nav-link text-light" href="#Schedule">Schedule</a>
+                </li>
+            </ul>     
+        </div>  
+    </nav>
+    <div class="container">
+        <h1 class="text-center">Girl Develop It</h1>
+        <!-- Primera sección: About -->
+        <h2 id="About">About</h2>
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/index.html]: Try to always use kebab-case for classes and id's, which is always lower case

File: Exercise-HTML-CSS-1/index.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+        <p>
+        It can be intimidating for women to learn and ask questions when they are in an <strong>extreme minority.</strong> While opn and welcoming, today's budding developer community is up to <strong>91% male.</strong> If we can empower more females with the confidence in their technological capabilities <em>we can begin to change this</em>
+        </p>
+        <hr>
+        <div class="row">
+            <div class="col-md">
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/index.html]: Nice touch! Seems that you understand grid system

File: Exercise-HTML-CSS-1/index.html (hunk 2 above)
Comment (sgdiosdado) [reply Exercise-HTML-CSS-1/index.html]: Will do!

**Pull request closed without merging**